File: Python/bmiimperial.py
def bmi_imperial(weight, height):
    weight_kilos = weight * 0.453592
    height_meters = height * 0.0254
    return weight_kilos/(height_meters**2)


if __name__ == "__main__":
    w = float(input("Please enter weight in pounds: "))
    h = float(input("Please enter height in inches: "))
    bmi = bmi_imperial(w, h)
    print(f"BMI is: {bmi}")

File: C++/bmiimperial.cpp
#include <iostream>
#include <iomanip>

int main() {
    double weight, height, weightInKilos, heightInMeters, bmi;

    std::cout << "Please enter weight in pounds: ";
    std::cin >> weight;
    std::cout << "Please enter height in inches: ";
    std::cin >> height;

    weightInKilos = weight * 0.453592;
    heightInMeters = height * 0.0254;
    bmi = weightInKilos/(heightInMeters*heightInMeters);
    std::cout << std::setprecision(2) << std::fixed << "BMI is: " << bmi;
    return 0;
}
